docs/functions.md

```diff
@@ -19,6 +19,9 @@ Currently we have almost 15+ functions in our npm package. Some of them are list
     * [doMod()](aFunc/doMod.md):- Get the modulus
 
 - Miscellaneous Functions
+    * [doMax()](mFunction/doMax.md):- Maximum element of Array
+    * [doMin()](mFunction/doMin.md):- Minimum element of Array
+    * [doFactorial()](mFunction/doFactorial.md):- Factorial of a number

```

Comment:
Change the `mFunction` tp `mFunc` here. Otherwise, docsify won't parse it to display it on the website.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
Fixed that issue. Please review